framework/wazuh/core/indexer/commands.py

```diff
@@ -1,13 +1,13 @@
 from dataclasses import asdict
-from typing import List, Optional
+from typing import List, Optional, Union
 
-from opensearchpy import NotFoundError
 from uuid6 import UUID
 
 from .base import BaseIndex, IndexerKey, remove_empty_values
-from wazuh.core.indexer.models.commands import Command, Status
 from wazuh.core.exception import WazuhResourceNotFound
+from wazuh.core.indexer.models.commands import Command, Result, Status
 
+ID_KEY = 'id'
```

Comment:
Is the value related to the id of a document or the literal value `id` that can be found in some keys of the Indexer responses?

Reply:
It is related to the document ID, the other one starts with an underscore (`_id`)